conda/base/context.py

```diff
@@ -111,6 +111,7 @@ class Context(Configuration):
     notify_outdated_conda = PrimitiveParameter(True)
     clobber = PrimitiveParameter(False)
     changeps1 = PrimitiveParameter(True)
+    prompt = PrimitiveParameter("({default_env}) ")
```

Comment:
I think I originally named it `prompt_modifier`.  But then I realized there was going to be interaction with the `CONDA_PROMPT_MODIFIER` environment variable that we're already setting in `activate.py`.  Wonder if we could back that out since it was only introduced in conda 4.4...

Reply:
`env_prompt` might be a bit more descriptive. That would create a `CONDA_ENV_PROMPT` environment variable -- something without a name collision, but also something with a `CONDA_ENV` prefix. Would that confuse people since it's actually not `conda-env`-related? Don't know. Then again, the audience of `CONDA_*` configuration variables are "power users".

Reply:
> Would that confuse people since it's actually not `conda-env`-related? 

Maybe. But over the next year it's my guess that a lot of the `conda env` subcommand stuff slowly fades away as we unify the environment specs between conda, conda-env, and anaconda-project.  So I'm ok with "maybe".